### ISSUE TITLE
First time installation is not clear 

### ISSUE THREAD

**Issue by alanedelman** [JuliaEditorSupport/julia-vscode#270]
II gather this is written for vscode users but if you've never used vscode it's
hard to follow.  

I gather one must find the extensions icon on the left and search for julia.

Also it's not intuitive that one edits in the right side of the pane as the defaults are on the left.
Should there be two sets of instructions?

Also it's not easy to figure out how to use the bottom right to tell the thing that it's a julia file.

The goal is to get to 1+1 and plot(sin,0,10) as easy as possible as a newbie.

**Comment by ZacLN** [comment]
Yep we're a bit behind with documentation. 

For documentation on basic vscode use I'll point you towards their [documentation](https://code.visualstudio.com/docs) -and specifically for extension installation, [this](https://code.visualstudio.com/docs/editor/extension-gallery))- as I don't think thats really our domain.


We're going to put out a new version this week and I'll have a big update of the docs for that release.

**Comment by ArchieCall** [comment]
@alanedelman I came over from Atom and find VS Code is snappier and yields an overall better impression  on the user.

**Comment by alanedelman** [comment]
It's not just the extensions, it's the combination of pieces that is hard to understand.
Here is what seems to work, though my images didn't copy.
I can try to send them over if useful, or perhaps they are obvious.

1. download and open up with this icon:
Inline image 1

2. Press the extensions button (bottom one in the sequence below)
Inline image 2
3.  search for julia
Inline image 3

4. hit the install button for julia language support
5. Code-->Preferences--->settings
place your julia path on the right
Inline image 4
and file-->save

6. File--->new file
click on the bottom right:

Inline image 5
then search above for julia:
Inline image 6
and select

7. then with control+enter executing julia works:
Inline image 7

**Comment by davidanthoff** [comment]
@alanedelman Let us know if the new README is good enough or not!
